business-central-parent/business-central-webapp/pom.xml

```diff
@@ -2183,6 +2183,15 @@
       </testResource>
     </testResources>
 
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>com.github.spotbugs</groupId>
+          <artifactId>spotbugs-maven-plugin</artifactId>
+        </plugin>
+      </plugins>
+    </pluginManagement>
+
```

Comment:
Why we have this here, isn't it duplicating what is in [kiegroup/droolsjbpm-build-bootstrap#921](https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/921) ?